src/ansys/tools/path/path.py

```diff
@@ -570,15 +570,15 @@ def save_mapdl_path(exe_loc=None, allow_prompt=True) -> str:
 
     Notes
     -----
-    The configuration file location (``config.txt``) can be found in
-    ``appdirs.user_data_dir("ansys_tools_path")``. For example:
```

Comment:
@germa89 Pre-existing - but this helpstring does not quite match the purpose of `save_mapdl_path`